contracts/veBalFeeInjector.sol

```diff
@@ -57,15 +61,17 @@ contract veBalFeeInjector is ConfirmedOwner, Pausable {
     whenNotPaused
     returns (bool upkeepNeeded, bytes memory performData)
   {
+    int counter = 0;
```

Comment:
Moved to using this counter concept to assert that all tokens were present instead of at least one.